convene-web/config/environments/test.rb

```diff
@@ -12,7 +12,7 @@
   # Do not eager load code on boot. This avoids loading your whole application
   # just for the purpose of running a single test. If you are using a tool that
   # preloads Rails for running tests, you may have to set it to true.
-  config.eager_load = false
+  config.eager_load = true
```

Comment:
Let me know if the solution in https://github.com/zinc-collective/convene/pull/172 looks good to you.
It can be merge as a separate PR or just merged into this branch.